src/components/page/page.comp.tsx

```diff
@@ -47,23 +47,23 @@ const buildInitQueryParamsAndPaginationState = (initQueryParams: any[], paginati
   if (paginationConfig) {
     initQueryParams.push({
       name: paginationConfig?.params?.page?.name,
-      label: 'Page',
+      label: paginationConfig?.params?.page?.label || 'Page',
       value: initialPagination?.page
     });
 
     if (paginationConfig?.params?.limit) {
       initQueryParams.push({
         name: paginationConfig?.params?.limit.name,
-        label: 'Limit',
+        label: paginationConfig?.params?.limit.label || 'Limit',
```

Comment:
Missing an optional operator on "limit".

Reply:
You mean the `?`? Actually it's the other way around, there should be no question mark at all and be `paginationConfig.params.limit.label` since the `if` just above checks everything out. Typescript wouldn't allow it if there was a question mark missing.

Reply:
Got it. Sorry, I didn't see it.